tests/vmi_lifecycle_test.go

```diff
@@ -1281,6 +1281,7 @@ var _ = Describe("[rfe_id:273][crit:high][vendor:cnv-qe@redhat.com][level:compon
 		Context("VirtualMachineInstance Emulation Mode", decorators.SoftwareEmulation, func() {
 
 			It("[test_id:1643]should enable emulation in virt-launcher", func() {
+				vmi = libvmifact.NewAlpine()
```

Comment:
TBH I don't understand how the situation described in the PR description is physically achievable;
Each parallel ginkgo process is running 1 spec at a time and has it's own dedicated `vmi` variable

I **think** this test is always skipped on environments that don't have `kv.Spec.Configuration.DeveloperConfiguration.UseEmulation` set to true

Reply:
Sorry, was looking at an old branch. It USED to always get skipped, now, you have to use the label filter with `!software-emulation` to skip it. I think it's basically getting skipped in all lanes in kubevirt.

Since the test is so simple maybe we could just convert it to a unit test?

Reply:
Apprently this is a hot topic https://github.com/kubevirt/kubevirt/pull/13722#issuecomment-2595463090

Reply:
As @akalenyu said, this test will soon be removed.